frontend/src/ManageOrgUsers/ManageOrgUsers.jsx

```diff
@@ -36,6 +37,9 @@ class ManageOrgUsers extends React.Component {
       this.setState({
         users: data.users,
         isLoading: false,
+        currentUserDetails: data.users.find(
```

Comment:
Works only if we fetch all the users in one request. Once we add pagination to this page, it won't work. This solution cannot be accepted.